pkgs/applications/misc/streamdeck-ui/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, python3Packages
+, wrapQtAppsHook
+, writeText
+, makeDesktopItem
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "streamdeck-ui";
+  version = "2.0.4";
+
+  src = python3Packages.fetchPypi {
+    pname = "streamdeck_ui";
+    inherit version;
+    sha256 = "650bef0da4957e0d2e2b4cd9c74a55715bcf1526d9ef2cdd04271364117a5845";
+  };
+
+  # Add udev rules to allow non-root access to the Stream Deck.
+  # To enable these rules on NixOS, add this to your system configuration:
+  #
+  #     services.udev.packages = [ pkgs.streamdeck-ui ];
```

Comment:
That should either be a module or be in the longDescription so that it can be easily found.

Reply:
This is added by the module, so I removed the comment.